src/tests/wscript

```diff
@@ -1,41 +1,218 @@
 #!/usr/bin/env python
 # encoding: utf-8
 
+import subprocess
+import os
+
+all_combinations_init_code = """#include <boost/fusion/container/vector.hpp>
```

Comment:
Isn't it better to put this generation of tests logic (string, func, etc) into a separate file and only use one function in the `wscript`? The `wscript` would be cleaner.